Add factory reset chapter (#8)
NethServer/nethsecurity-docs: reset_recovery.rst

to a configuration error, failsafe mode comes to your rescue. When you boot into failsafe mode, the device initializes in a basic operating state,
using a set of predefined defaults, allowing you to address the issue manually.

However, it's important to note that failsafe mode cannot resolve more complex issues such as faulty hardware or a damaged kernel.
While it resembles a reset, failsafe mode enables you to access your device and restore settings if necessary, whereas a reset would simply erase all configurations.

The simplest way to activate failsafe mode is to connect directly to the firewall using a VGA/HDMI monitor or a serial cable. To do this,
boot the machine, wait for the Grub boot menu to appear, and select `NethSecurity (failsafe)`.

You can access the firewall directly through the serial port using a null-modem cable and a common terminal program.
For Windows, you can use PuTTY version 0.60 or higher. For Linux, options include minicom and picocom. Set the baud rate
to 115200, data bits to 8, parity to None, and stop bits to 1 (abbreviated as 8N1).

> You can access the firewall directly through the serial port using a null-modem cable and a common terminal program (including PuTTY, version >= 0.60).

Reviewer: Repetition: delete 2 two lines.

Author: Done

Set the baud rate to 115200, data bits to 8, parity to None, and stop bits to 1 (abbreviated as 8N1).

> After entering failsafe mode, the firewall will initiate with a network address of 192.168.1.1/24, usually on the eth0 network interface,

Reviewer: ```suggestion
After entering failsafe mode, the firewall will start with a network address of 192.168.1.1/24, usually on the eth0 network interface,
```

and only essential services will be operational. It's important to note that the DHCP server will be inactive in failsafe mode.
Follow the instructions displayed on the screen to mount the root filesystem and access other utilities as needed.

.. _recovery-section:

Emergency recovery
==================

> Emergency recovery in NethSecurity, also known as un-bricking, is a  feature enabling users to restore their firewall device in cases of severe malfunctions.

Reviewer: ```suggestion
Emergency recovery in NethSecurity, also known as unbricking, is a feature enabling users to restore their firewall device in cases of severe malfunctions.
```

Unbricking ensures that even the most critical issues can be resolved, restoring the device to full functionality, unless there are hardware failures.

If you still have access to the system, you can use the following commands to download and write the image: ::

>   ns-download-l

Reviewer: ```suggestion
  ns-download -l
```
